connectivity/check/context.go

```diff
@@ -383,6 +383,16 @@ func (ct *ConnectivityTest) Run(ctx context.Context) error {
 		ct.Failf("writing to junit file %s failed: %s", ct.Params().JunitFile, err)
 	}
 
+	if ct.Params().FlushCT {
+		for _, ciliumPod := range ct.CiliumPods() {
```

Comment:
This can be done concurrently, similar to https://github.com/cilium/cilium-cli/blob/main/bgp/peers.go#L67-L78

Reply:
Aye, good suggestion. Currently, our CI K8s clusters consist of ~4 nodes, so the parallelism was never a concern. However, we have plans to run those tests on way larger clusters.